yt_dlp/YoutubeDL.py

```diff
@@ -2641,6 +2641,19 @@ def _fill_common_fields(self, info_dict, final=True):
             if final and info_dict.get('%s_number' % field) is not None and not info_dict.get(field):
                 info_dict[field] = '%s %d' % (field.capitalize(), info_dict['%s_number' % field])
 
+        multivalue_fields = {
+            'album_artist': 'album_artists',
+            'artist': 'artists',
+            'composer': 'composers',
+            'creator': 'creators',
+            'genre': 'genres',
+        }
+        for old_key, new_key in multivalue_fields.items():
+            if old_value := info_dict.get(old_key):
+                info_dict[new_key] = re.split(r', ?', old_value)
+            elif new_value := info_dict.get(new_key):
+                info_dict[old_key] = ', '.join(new_value)
```

Comment:
We could perhaps do (remove space):
```suggestion
                info_dict[old_key] = ', '.join(re.replace(', +', ',', v) for v in new_value)
```
or maybe (unicode):
```suggestion
                info_dict[old_key] = ', '.join(v.replace(', ', '， ') for v in new_value)
```

as partial workaround to https://github.com/yt-dlp/yt-dlp/issues/3944#issuecomment-1686854709 for those who don't have mutagen

Opinions?

We don't have to do it in this PR. Just asking

Reply:
I think the second one (unicode) is the better solution.